pkg/common/policy/policy_examples.go

```diff
@@ -0,0 +1,349 @@
+package policy
+
+const defaultPermissionData = `
```

Comment:
It's a little weird for server-specific defaults to live under `common`. I wonder if there shouldn't be a `pkg/server/policy` package that has a `NewEngineOrDefault()` method that either constructs an engine with the provided config or returns an engine with the default policy loaded. That would also get rid of my concern about NewEngine returning `nil`.